devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
It seems that this setup package is using php5. I've been looking into other options.